### ISSUE TITLE
CV outputs 3d point layer

### ISSUE THREAD

**Issue by emlys** [natcap/invest#749]
Allison reported that a CV output layer geometry was the Point Z (3D) type, which is unnecessary and troublesome because she needed to convert it back to 2D.

**Comment by davemfish** [comment]
I see `Point` for the geometry of `coastal_exposure.gpkg`

```
λ ogrinfo -al -so coastal_exposure.gpkg
INFO: Open of `coastal_exposure.gpkg'
      using driver `GPKG' successful.

Layer name: coastal_exposure
Geometry: Point
```

**Comment by davemfish** [comment]
```python
from osgeo import ogr

point_list = [(0, 0), (1, 1), (2, 2)]

for point in point_list:
    print(point)
    geometry = ogr.Geometry(ogr.wkbPoint)
    geometry.AddPoint(*point)
    print(geometry)
```
```
(0, 0)
POINT (0 0 0)
(1, 1)
POINT (1 1 0)
(2, 2)
POINT (2 2 0)
```

This is a minimal example of what CV is doing. I believe we're using the `ogr` API correctly, but nevertheless we end with the z-dimension.

**Comment by davemfish** [comment]
Also I think we need an example of how this causes a problem, otherwise I might suggest it's not a bug.

**Comment by davemfish** [comment]
It looks like we can use `geometry.AddPoint_2D` to avoid this. And I guess that's the correct thing to do. https://github.com/OSGeo/gdal/blob/master/swig/python/osgeo/ogr.py#L5811

**Comment by davemfish** [comment]
Fixed in #759
